mlrun/api/utils/clients/iguazio.py

```diff
@@ -488,13 +488,38 @@ def _send_request_to_api(
             mlrun.errors.raise_for_status(response, error_message)
         return response
 
-    @staticmethod
     def _generate_auth_info_from_session_verification_response(
+        self,
         response: requests.Response,
     ) -> mlrun.api.schemas.AuthInfo:
-        gids = response.headers.get("x-user-group-ids")
-        if gids:
-            gids = gids.split(",")
+
+        (
+            username,
+            session,
+            planes,
+            user_unix_id,
+        ) = self._resolve_params_from_response_headers(response)
+
+        user_id, group_ids = self._resolve_params_from_response_body(response)
```

Comment:
These method names aren't really reflective of what they actually do. As `_resolve_params_from_response_body` looks at both headers and body.

Not sure how to rename them though. Maybe `_resolve_params_from_response_headers` can stay as it is but also get the `user_id` and `group_ids` (which might be `None`) and `_resolve_params_from_response_body` can get those only from the body and replace those vars? WDYT? Or different method names.